.github/workflows/tag-workflow.yaml

```diff
@@ -38,29 +38,19 @@ jobs:
     - name: Docker login
       run: docker login -u $DOCKER_USER -p $DOCKER_TOKEN
 
-      # Build the Scalable Pixel Streaming Frontend Docker image from the dist directories of the packages
-    - name: Build the Scalable Pixel Streaming Frontend Docker image for AWS and push to Docker
+      # Set up Node js for npm
+    - name: Setup Nodejs
+      uses: actions/setup-node@v3
+     
+      # Build the SPS Frontend 
+    - name: Build SPS Frontend
       run: |
-        ./.scripts/linux/pipeline/build-images-core.sh --version $FRONTEND_VERSION --provider $CLOUD_PROVIDER
-
-  # Build CoreWeave
-  build-coreweave:
-
-    runs-on: ubuntu-latest
-
-    env:
-      CLOUD_PROVIDER: coreweave
-
-    steps: 
-
-    # Clone the source code on the runner
-    - name: Clone repository
-      uses: actions/checkout@v3
-
-      # Login to Docker
-    - name: Docker login
-      run: docker login -u $DOCKER_USER -p $DOCKER_TOKEN
+        cd /Frontend/examples/typescript
```

Comment:
The `cd` will fail as it's attempting to cd into the `Frontend` directory at the root of the runner. For this to work, it would need to be `cd  ./Frontend/examples/typescript`

Additionally, since this workflow's purpose is to build the library and not the entire frontend, we should be ne running our `npm run build` script from the `library` directory.

Reply:
When all is said and done and all parts are moved in this will change. This is a place holder atm.